guides/source/action_view_overview.md

```diff
@@ -118,93 +157,40 @@ would produce something like:
-maintained by the Rails team and included in the default Rails `Gemfile`.
-It's similar to Builder but is used to generate JSON, instead of XML.
-
-If you don't have it, you can add the following to your `Gemfile`:
-
-```ruby
-gem "jbuilder"
-```
-
-A Jbuilder object named `json` is automatically made available to templates with
-a `.jbuilder` extension.
-
-Here is a basic example:
-
-```ruby
-json.name("Alex")
-json.email("alex@example.com")
-```
-
-would produce:
-
-```json
-{
-  "name": "Alex",
-  "email": "alex@example.com"
-}
-```
-
-See the [Jbuilder documentation](https://github.com/rails/jbuilder#jbuilder) for
-more examples and information.
+See [Builder documentation](https://github.com/jimweirich/builder) for more examples.
 
-### Template Caching
+### Caching
 
 By default, Rails will compile each template to a method to render it. In the development environment, when you alter a template, Rails will check the file's modification time and recompile it.
 
+There is also Fragment Caching for when different parts of the page need to be cached and expired separately. Learn more about it in the [caching guide](caching_with_rails.html#fragment-caching).
+
 Partials
 --------
 
-Partial templates - usually just called "partials" - are another device for breaking the rendering process into more manageable chunks. With partials, you can extract pieces of code from your templates to separate files and also reuse them throughout your templates.
+Partial templates - usually just called "partials" - are a way of breaking up the view templates into smaller reusable chunks. With partials, you can extract a piece of code from your main template to a separate smaller file, and render that file in the main template. You can also pass data to the partial files from the main template.
+
+Let's see this in action with some examples:
 
 ### Rendering Partials
 
-To render a partial as part of a view, you use the `render` method within the view:
+To render a partial as part of a view, you use the [`render`](https://api.rubyonrails.org/classes/ActionView/Helpers/RenderingHelper.html#method-i-render) method within the view:
 
 ```erb
-<%= render "menu" %>
+<%= render "product" %>
 ```
 
-This will render a file named `_menu.html.erb` at that point within the view that is being rendered. Note the leading underscore character: partials are named with a leading underscore to distinguish them from regular views, even though they are referred to without the underscore. This holds true even when you're pulling in a partial from another folder:
+This will look for a file named `_product.html.erb` on the same folder to render within that view. Partial file names start with leading underscore character by convention. The file name distinguishes partials from regular views. However, no underscore is used when referring to partials for rendering within a view. This is true even when you reference a partial from another directory:
```

Comment:
```suggestion
This will look for a file named `_product.html.erb` in the same folder to render within that view. Partial file names start with leading underscore character by convention. The file name distinguishes partials from regular views. However, no underscore is used when referring to partials for rendering within a view. This is true even when you reference a partial from another directory:
```